vite.config.ts

```diff
@@ -10,9 +10,6 @@ export default defineConfig({
     tsconfigPaths({
       projects: ['./tsconfig.json']
     }),
-    eslint({
-      fix: true
-    }),
```

Comment:
I know it's a bit annoying but I'm not sure if we should remove this. On the one hand, we use `husky` to check for `eslint` issues before making a commit, so maybe it's not necessary. On the other hand, it makes it easier to detect problems in code.